src/icons/assignee-placeholder.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="18" height="20" viewBox="0 0 13 12" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
The width and height of the svg images are inconsistent. I think it should be 12*12 according to Figma. This is producing inconsistent behaviour as shown below - 

![Screenshot from 2024-07-05 15-11-14](https://github.com/copilot-platforms/tasks-app/assets/38468429/8579eaff-831c-4a2c-a03a-0b0b3cf2fd80)